src/interface/meta.thrift

```diff
@@ -291,6 +319,63 @@ struct HBReq {
     1: common.HostAddr host,
 }
 
+struct CreateUserReq {
+    1: UserItem user,
+    2: string encoded_pwd,
+    3: bool missing_ok,
```

Comment:
What's this?

Reply:
> What's this?

missing_ok？ If the syntax contains `IF NOT EXISTS` ,  missing_ok should be true, else should be false.